### ISSUE TITLE
tmpfs broken on 20160901T054050Z?

### ISSUE THREAD

**Issue by sjorge** [joyent/illumos-joyent#129]
```bash
root@artemis:~# mount /data/plex/Logs/
mount: mount tmpfs on /data/plex/Logs failed: Bad address
```

```bash
root@artemis:~# grep tmpfs /etc/fstab
tmpfs  		/data/plex/Logs	tmpfs  	rw,size=64m,mode=0750,uid=plex,gid=plex	0 0
```
It works fine on 20160818T234814Z as reverting fixes the error/

**Comment by sjorge** [comment]
I narrowed it down to these mount options ```uid=plex,gid=plex```

**Comment by sjorge** [comment]
As pointed out by bahamat, https://smartos.org/bugview/OS-4710

**Comment by jjelinek** [comment]
I opened OS-5644 for this

**Comment by jjelinek** [comment]
I've tested a fix and it is up for review.

**Comment by sjorge** [comment]
Let me know once one of the builds with it hit the joyent-dev location, I can snag it up and give it a spin.

**Comment by jjelinek** [comment]
The fix has been pushed in commit 12a530b and should be available in a build later today.

**Comment by sjorge** [comment]
All good in one of the master release having that commit, will upgrade to the new released PI tomorrow.
